modules/op-running-pipeline-and-task-run-pods-with-privileged-security-context.adoc

```diff
@@ -0,0 +1,69 @@
+[id='op-running-pipeline-and-task-run-pods-with-privileged-security-context']
+= Running pipeline run and task run pods with privileged security context
+:context: op-running-pipeline-and-task-run-pods-with-privileged-security-context
+
+toc::[]
+
+.Procedure
+
+To run a pod (resulting from pipeline run or task run) with `privileged` security context, do the following modifications:
+
+* Configure the associated user account or service account to have an explicit SCC. You can perform the configuration using any of the following methods:
+** Use the following OpenShift command:
++
+[source, terminal]
+----
+$ oc adm policy add-scc-to-user <scc-name> -z <service-account-name>
+----
+** Manually modifying the YAML for `RoleBinding` and `Role` or `ClusterRole`:
+
++
+.Example `RoleBinding` object
+[source,yaml,subs="attributes+"]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: service-account-name <1>
+  namespace: default
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: pipelines-scc-clusterrole <2>
+subjects:
+- kind: ServiceAccount
+  name: pipeline
+  namespace: default
+
+----
+<1> Substitute with the same service account name as used in the `oc` command above.
```

Comment:
Based on the doc, the way I understood it - You can configure your SA either by running the oc command or (not 'and'. If we need to use the command as well as modify YAMLs then we will need to use a numbered list) by modifying the YAMLs. If yes, then let us not link the call out with the command - Change: 'Substitute with the same service account name as used in the `oc` command above.' to 'Substitute with an appropriate service account name.'

Reply:
Good catch. Done!